cmake/nlohmann-json.cmake

```diff
@@ -26,7 +26,7 @@ ExternalProject_Add(nlohmann_json_download
 )
 
 ExternalProject_Get_Property(nlohmann_json_download INSTALL_DIR)
-SET(NLOHMANN_JSON_INCLUDE_DIR ${INSTALL_DIR}/third_party/src/nlohmann_json_download/single_include)
+SET(NLOHMANN_JSON_INCLUDE_DIR ${INSTALL_DIR}/src/nlohmann_json_download/single_include)
```

Comment:
Wondering why `third_party` should be removed as it was declared as PREFIX in line 6 which is the download root?

Reply:
To my understanding exactly for that reason.

Since PREFIX is specified, all the other directories associated with the external project will be created under it (per CMake docs).

Thus, the install directory (which is used to obtain the include directory) already contains it.

Reply:
I see, thanks for the fix.